### DISCUSSION TITLE
Which parameter values with which model?

### DISCUSSION THREAD

**Discussion by lappemic** [pybamm-team/PyBaMM#4013] [Q&A] [answered]
Does anyone know if there's a comprehensive guide detailing which models are compatible with specific parameter values in PyBaMM? Also, which parameters are adjustable for each model?

**Comment by valentinsulzer** [comment, marked as answer]
The review articles here are a good starting point: https://docs.pybamm.org/en/stable/source/user_guide/fundamentals/battery_models.html

You can also run some "trial and error" code to see if any single model / parameter combo works, see: https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/print_model_parameter_combinations.py
